### ISSUE TITLE
Conjugation() doesn't return proper tense

### ISSUE THREAD

**Issue by turbobuilt** [spencermountain/compromise#601]
```javascript  
var doc = nlp("it was cold");
console.log(doc.verbs().conjugation())
```
Outputs
```
[ 'Present' ]
```
Should output something about past.

**Comment by spencermountain** [comment]
holy, i just looked and `was` has been missing this tag since at least January 😰.

thank you. There's some 4-thousand tests. It's wild that this has happened.

fixed in `11.14.1`.

**Comment by turbobuilt** [comment]
Yeah, I was kind of surprised myself!  But it happens to the best of us.  Thanks for fixing it!
